gateway/src/commonMain/kotlin/DefaultGateway.kt

```diff
@@ -179,21 +179,11 @@ public class DefaultGateway(private val data: DefaultGatewayData) : Gateway {
         }
     }
 
-    private fun Frame.deflateData(): String {
-        val outputStream = ByteArrayOutputStream()
-        InflaterOutputStream(outputStream, inflater).use {
-            it.write(data)
-        }
-
-        return outputStream.use {
-            String(outputStream.toByteArray(), 0, outputStream.size(), Charsets.UTF_8)
-        }
-    }
-
     private suspend fun read(frame: Frame) {
+        defaultGatewayLogger.trace { "Received raw frame: $frame" }
```

Comment:
hm, should we add additional logging in the scope of this mpp pr?

Reply:
or rather: why did you add this?

Reply:
For additional debugging abilities of decompression. It might be smart to add this, as we know have to maintain two compression implementations

Reply:
sure 👍